File: 백준/Bronze/9498. 시험 성적/시험 성적.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
})

rl.question('', (inputs) => {
    const [a] = inputs.split(' ').map(Number);

    my_function(a);

    rl.close();
})

let my_function = (a) => {
    if(a >= 90){
        console.log('A');
    } 
    else if(a >= 80){
        console.log('B');
    }
    else if(a >= 70){
        console.log('C');
    }
    else if(a >= 60){
        console.log('D');
    }
    else{
        console.log('F');
    }
}

File: 백준/Bronze/9498. 시험 성적/시험 성적.py
import sys
input = sys.stdin.readline








if __name__ == "__main__":
    score = int(input())

    if(score > 89):
        print("A")
    elif(score > 79):
        print("B")
    elif(score > 69):
        print("C")
    elif(score > 59):
        print("D")
    else:
        print("F")
